.github/workflows/wasm.yml

```
@@ -120,7 +120,7 @@ jobs:
 
   test:
     needs: [build-wasm, build-nargo]
-    name: Test
+    name: Test wasm
```

```suggestion
    name: Test noir_wasm
```

We're going to be having more wasm crates in this repository so I'd like to be specific here.